random_name.js

```diff
@@ -0,0 +1,31 @@
+/* Develop function that will handle random name generations.
+
+The function will return a randomly generated name, that is defined as ```description``` + ```food```, similar to the feature found on Google Docs.
+
+Examples
+```js
+generateRandomName () // Fried Cake
+generateRandomName () // Soggy Drumstick
+generateRandomName () // Crispy Noodle
+```
+
+Ideally, it should be designed in a way where we can easily change the available keywords
+*/
+
+function generateRandomName() {
```

Comment:
Since we need to use this function, we need to export this function.

Reply:
done